### DISCUSSION TITLE
Can't validate local image folders that have a ":" on Windows

### DISCUSSION THREAD

**Discussion by Blop90** [gaddlord/mtg.studio#392] [Q&A] [answered]
I started setting up a local image database with high-res images I downloaded somewhere.
MTG Studio is looking for 352 Edition folders, but some of them must have the ":" folder name.
Since there can't be a folder name with a : on Windows I can't import these perticular Editions, see screenshot.

Is there a workaround?

![7vZHJbWqP9](https://user-images.githubusercontent.com/105953901/185751694-0cb4ca2d-2e06-4c66-9464-7e3af3d46169.png)

**Comment by gaddlord** [comment, marked as answer]
Replied in https://github.com/gaddlord/mtg.studio/issues/393
